README.md

```diff
@@ -33,7 +33,9 @@ The build tools also include a CLI, which can be used by installing the tools gl
 | ----------------------- | ----------- | -------------------------------------------------------------------------- |
 | `destination` (*)       | `string`    | Target location for the Material build.                                    |
 | `modules`               | `string[]`  | Modules that should be part of the build.<br/> All modules will be built if nothing is specified.                                                                                                           |
-| `version`               | `string`    | Version of AngularJS Material.<br/> If set to `local`, it will take the local installed AngularJS Material version from the node modules. <br/> If set to `latest`, the latest version will be downloaded.                                                           |
+| `excludeModules`               | `string[]`  | Modules that should be excluded from the build.<br/> Use to exclude e.g. core modules when building a bundle that should get lazy-loaded.                                                                                                           |
```

Comment:
We should mention that this is not a full string match, but it also matches partially. maybe an example as you already added in code? (`animate` => `material.core.animate`)

Reply:
I agree that this would be a helpful addition as well.